### DISCUSSION TITLE
how can I create sync.Context?I didn`t found the method

### DISCUSSION THREAD

**Discussion by mingmingshiliyu** [cschleiden/go-workflows#265] [Q&A]

**Comment by cschleiden** [comment]
`workflow.Background()` for example

**Comment by mingmingshiliyu** [comment]
> `workflow.Background()` for example

<img width="593" alt="image" src="https://github.com/cschleiden/go-workflows/assets/143821597/30811782-baf0-40ba-ad06-bd922a55ec3b">
none

**Comment by cschleiden** [comment]
Ah right, try `workflow.NewDisconnectedContext(ctx)` 

For unit testing workflows, try the built-in workflow tester. It will automatically provide the right context to your workflow.

**Reply by mingmingshiliyu** [reply]
I got it
